addons/flashlights/CfgVehicles.hpp

```diff
@@ -0,0 +1,45 @@
+class CfgVehicles {
+    class Man;
+    class CAManBase: Man {
+        class ACE_SelfActions {
+            //todo: add flashlight attach actions
+        }; 
+    };
+    
+    class NATO_Box_Base;
+    class EAST_Box_Base;
+    class IND_Box_Base;
+    class FIA_Box_Base_F;
+
```

Comment:
@gienkov did you watch the youtube vid?
https://www.youtube.com/watch?v=q8pVBif9KYU